src/sentry/static/sentry/app/components/contextData.jsx

```diff
@@ -208,16 +208,9 @@ class ContextData extends React.Component {
   };
 
   render() {
-    // XXX(dcramer): babel does not support this yet
-    // let {data, className, ...other} = this.props;
-    const data = this.props.data;
-    const className = this.props.className;
-    const other = {};
-    for (const key in this.props) {
-      if (key !== 'data' && key !== 'className') {
-        other[key] = this.props[key];
-      }
-    }
+    // We do not use preserveQuotes, but require it to be removed from ...other
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Instead of this, let's explicitly ignore `preserveQuotes` by prefixing it with an underscore, e.g.

```
    const {data, className, preserveQuotes: _preserveQuotes, ...other} = this.props;
```

Reply:
oh sweet wasn't sure how to do this thanks!